Server/MarkerEditorConfig.js

```diff
@@ -277,7 +277,7 @@ class MarkerEditorConfig extends ConfigBase {
             this.#host = '0.0.0.0';
             this.#port = 3232;
         } else {
-            this.#dataPath = this.#getOrDefault('dataPath', MarkerEditorConfig.getDefaultPlexDataPath);
+            this.#dataPath = this.#getOrDefault('dataPath', MarkerEditorConfig.getDefaultPlexDataPath());
```

Comment:
Sorry, didn't see this PR until I already pushed my own fix (107ea2be1ce36b4aaf42049051ce54a8b1f7be47).

The removal of `()` was intended here though to avoid checking potential data paths/accessing Windows's registry until we know we want to use the default value. The issue is that I didn't also update `ConfigBase.#checkType` to handle the case where the `defaultValue` was a function that had to be called, so it complained about the type mismatch.